packages/dev/core/src/Misc/fileTools.ts

```diff
@@ -671,6 +673,16 @@ export const IsBase64DataUrl = (uri: string): boolean => {
     return Base64DataUrlRegEx.test(uri);
 };
 
+export const TestBase64DataUrl = (uri: string): { match: boolean; type: string } => {
+    const results = Base64DataUrlRegEx.exec(uri);

```

Comment:
Can you replace with IsBase64DataUrl ?

Reply:
should it also check for length ?

Reply:
I created a new method because besides testing for the regex I need the value of it, which exec returns